tests/integration/test_github_actions.py

```diff
@@ -0,0 +1,20 @@
+"""A test that is run only by Github Actions
+
+This test makes real network requests, so environment variables
+should be specified in Github Actions.
+"""
+import os
+
+import pytest
+
+from praw import Reddit
+from praw.models import Submission
+
+
+@pytest.mark.skipif(
+    "true" not in os.environ.get("GITHUB_ACTIONS", "").lower(),
```

Comment:
I think this can be simplified to just getting the value of `GITHUB_ACTIONS`. It shouldn't be set if we're not on GITHUB_ACTIONS.